src/Components/Payment/PaymentFormWrapper.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react"
+import { RelayProp } from "react-relay"
+import { Elements, StripeProvider } from "react-stripe-elements"
+import PaymentForm from "./PaymentForm"
+
+declare global {
+  interface Window {
+    Stripe?: (key: string) => stripe.Stripe
+    sd: {
+      STRIPE_PUBLISHABLE_KEY: string
+    }
+  }
+}
```

Comment:
https://github.com/artsy/reaction/blob/master/typings/sharify.d.ts#L18

Reply:
I updated  this to just  pull in `sd` directly. I'm actually not sure why it was extracted  separately before, but this appears to work?

Reply:
Not sure? I think `STRIPE_PUBLISHABLE_KEY` needs to go in the `sharify.d.ts` as well as the `webpack.config` file -- @williardx also recently had to update this for his logging work.

Reply:
updated